### ISSUE TITLE
AIF files version not being parsed

### ISSUE THREAD

**Issue by sblanky** [pauliacomi/pyGAPS#46]
**Describe the bug**
AIF file versions (from the webapp) are lately being given as a alphanumeric string;

```_audit_aif_version 6acf6ef```

This doesn't play nice with the AIF parser (`isotherm_from_aif()` in `pygaps.parsing.aif`), as it tries to force the the verion number, in this case `6acf6ef` to be a float. 

```py
 version = block.find_value('_audit_aif_version').strip("'")
 if not version or float(version.strip("'")) < float(_parser_version):
```
I don't know if this is meant to be hexadecimal or something, but the parser doesn't know what to do with it.

In my local version I've just deleted the version check!

**Comment by pauliacomi** [comment]
I see, there's been a subtle change in version numbering. The check will probably be removed.
I'll test and push this as soon as possible.

**Comment by pauliacomi** [comment]
I've dug a bit deeper, and it seems like the cause is a rewrite/update of the AIF format to change naming on some parameters. Not sure the way forward on this. It can be a re-write on the pyGAPS AIF parser but then some data might get lost. It might have to be a version-dependent AIF parser.

**Comment by jackevansadl** [comment]
pushed a branch to fix this such that pygaps works with the now widely released version of the aif dictionary.
https://github.com/jackevansadl/pyGAPS/tree/aifparserupdate

**Comment by pauliacomi** [comment]
Implemented and released in pyGAPS [v4.5.0](https://github.com/pauliacomi/pyGAPS/releases/tag/v4.5.0)

Thanks again for the heads-up!
